bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/component.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import { injectIntl } from 'react-intl';
+import _ from 'lodash';
+import UserListContent from './user-list-content/component';
+import UserAction from './user-action/component';
+
+const normalizeEmojiName = (emoji) => {
+  const emojisNormalized = {
+    agree: 'thumbs_up',
+    disagree: 'thumbs_down',
+    thumbsUp: 'thumbs_up',
```

Comment:
You have the same normalized strings in another file, please reuse, ideally from config file with consts